extensions/flac/src/main/jni/flac_parser.cc

```diff
@@ -191,6 +191,22 @@ void FLACParser::metadataCallback(const FLAC__StreamMetadata *metadata) {
         ALOGE("FLACParser::metadataCallback unexpected VORBISCOMMENT");
       }
       break;
+    case FLAC__METADATA_TYPE_PICTURE:
+      {
+        const FLAC__StreamMetadata_Picture *pic = &metadata->data.picture;
```

Comment:
nit - Avoid abbreviation. Maybe call this one `picture` and the one below `FlacPicture flacPicture`

Reply:
Done